docs/mobile-apps/virtual-usb.md

```diff
@@ -15,164 +15,247 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+## What You'll Need
 
->**NOTE**:Your access may depend on whether you're using the Legacy platform or the Sauce Labs platform. For more information on upcoming and available features for Real Devices, refer to the [feature preview doc](https://wiki.saucelabs.com/pages/viewpage.action?pageId=102721844).
+<p><Highlight color="#013a70">ENTERPRISE PLANS ONLY</Highlight></p>
 
-<img src={useBaseUrl('img/virtual-usb/vusb-workflow.png')} alt="Virtual USB Workflow" width="400" />
+* For security reasons, you'll need to have our [private real device cloud](https://saucelabs.com/platform/real-device-cloud) feature enabled as part of your [enterprise pricing plan](https://saucelabs.com/solutions/enterprise). This is a dedicated pool of Sauce Labs real devices allocated to your organization only.
+* Windows, Mac or Linux operating system.
+* Administrator and or installation rights on your machine.
+* Have Java Development Kit (JDK) installed.
+* If you want to use [Sauce Connect Proxy](https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy), have the client installed.
+* If you're testing Android apps:
+  * Android Debug Bridge (ADB) version higher than 1.0.39.
+  * Android Studio 4 or higher.
 
+## Virtual USB for Sauce Labs
 
-## What You'll Need
+<img src={useBaseUrl('img/virtual-usb/vusb-workflow.png')} alt="Virtual USB Workflow" width="800" />
 
-* You'll need to have our [private Real Device Cloud](https://saucelabs.com/platform/real-device-cloud) feature enabled as part of your plan. This is a dedicated pool of Sauce Labs real devices for your organization.
-* You'll need to have Java installed.
-* Android only:
-    * Make sure you're using an Android Debug Bridge (ADB) version higher than 1.0.39.
-    * Set `JAVA_HOME` and `ANDROID_HOME` as environment variables. To confirm, enter `echo %JAVA_HOME%`, and it will print that command. Then, do the same for `echo %ANDROID_HOME%`.
-    * Optional: Have Android Studio installed on your local machine, preferably the latest version.
+:::tip `--help` flag
+Run `java -jar virtual-usb-client-2.0.0.jar --help` to see a full list of vUSB test configuration commands and options.
+:::
 
-## vUSB for Sauce Labs (NEW)
+### Download Client
 
-Instructions for iOS and Android devices:
+1. Click below to download the latest vUSB client to your local machine where you have your IDE installed.
 
-:::tip Use `--help` to access instructions in the client
-Run `java -jar vusb-client.jar --help` to see a list of all available vUSB commands and options.
-:::
+  <p> <a href="https://s3-eu-west-1.amazonaws.com/v2.0.0/virtual-usb-client-2.0.0.jar"><button class="download">Download vUSB</button></a> </p>
 
-1. Click below to download the latest vUSB client on the same machine where you have Android Studio installed.
+>**NOTE**: If you're testing on iOS devices, you'll also need to [download iOS symbols](https://github.com/Zuikyo/iOS-System-Symbols).
```

Comment:
This is not 100% correct. The symbols will automatically be downloaded when you connect to a device. That should also be shown when using the latest vUSB runner, something like this

```log
11:13:12.341 [KQueueEventLoopGroup-2-2] INFO com.saucelabs.vusb.client.server.usbmuxd.UsbMuxServer - ********************************************************************************
11:13:12.341 [KQueueEventLoopGroup-2-2] INFO com.saucelabs.vusb.client.server.usbmuxd.UsbMuxServer - You may need to wait for some time if you connect to a device the first time.
11:13:12.341 [KQueueEventLoopGroup-2-2] INFO com.saucelabs.vusb.client.server.usbmuxd.UsbMuxServer - Xcode requires to download the debug symbols for each device that is connected
11:13:12.341 [KQueueEventLoopGroup-2-2] INFO com.saucelabs.vusb.client.server.usbmuxd.UsbMuxServer - to the host before the device can be used for developing. The debug symbols are
11:13:12.341 [KQueueEventLoopGroup-2-2] INFO com.saucelabs.vusb.client.server.usbmuxd.UsbMuxServer - stored in the folder `~/Library/Developer/Xcode/iOS DeviceSupport`. If Xcode
11:13:12.341 [KQueueEventLoopGroup-2-2] INFO com.saucelabs.vusb.client.server.usbmuxd.UsbMuxServer - finds the files for the attached device there it won't download them again.
11:13:12.341 [KQueueEventLoopGroup-2-2] INFO com.saucelabs.vusb.client.server.usbmuxd.UsbMuxServer - ********************************************************************************
```

Secondly, I don't think it's wise to refer to a 3rd party GitHub page for the symbols, if that person is stopping to maintain them, which is already the case, the repo is not useful.

Reply:
True for both -- thank you for clarifying, I was wondering about that. I will delete that sentence, since there's no action required. There is still a comment leftover in the old wiki doc. I'll reply to that & tag you both.


![Screen Shot 2021-04-07 at 11 18 42 PM](https://user-images.githubusercontent.com/56411016/113977753-e8992900-97f7-11eb-96fe-1bfbda1cdc82.jpg)